compiler/one-cmds/one-init

```diff
@@ -253,6 +326,25 @@ def _generate():
         config.write(f)
 
 
+def _get_model_type(parser, args):
+    if _utils._is_valid_attr(args, 'model_type'):
+        return args.model_type
+
+    if _utils._is_valid_attr(args, 'input_path'):
+        _, ext = os.path.splitext(args.input_path)
+
+        # ext would be, e.g., '.tflite' or '.onnx'.
+        # Note: when args.input_path does not have an extension, e.g., '/home/foo'
+        # ext after os.path.splitext() is '' and ''[1:] is still ''.
+        ext = ext[1:]
+        if ext in ["tflite", "onnx"]:
+            return ext
+        else:
+            parser.error(f'following file extensions are supported: ".onnx" ".tflite"')
```

Comment:
**FYI** These lines are added from #9619 to resolve https://github.com/Samsung/ONE/pull/9619#discussion_r956806115.